WalletWasabi.Fluent/Controls/Dialog.axaml

```diff
@@ -84,4 +85,19 @@
   <Style Selector="c|Dialog:not(:open) /template/ Panel#PART_Overlay">
     <Setter Property="Opacity" Value="0"/>
   </Style>
+
+  <!-- Dialog Fullscreen State -->
+  <Style Selector="c|Dialog[FullScreenEnabled=True] /template/ Border#PART_Border">
+    <Setter Property="Margin" Value="0 30 0 0" />
+    <Setter Property="BoxShadow" Value="0 0 0 0 #000000" />
```

Comment:
Maybe we can make this a resource, it can be useful in other places?

Reply:
The goal was here to reset the Border's boxshadow. I tried to set it to {x:Null}, but that caused NRE.
Do you know a better solution for it?
Good point, I will make a resource if there is no better solution.